lengthening-and-recoding.Rmd

```diff
@@ -0,0 +1,326 @@
+```
+
+## Pivoting Wider 
+
+This part will transform the data from long to wide for tableau
+
+Explicitly name the dimensions and measures 
+
+Define list of all columns names that are measures
+```{r Create measures/dimensions list}
+measures <- c(
+      "a4",
+      "aoj11",
+      "aoj12",
+      #"b0", Removed per Maita to Yilin on 4/21/2020
+      "b1",
+      "b12",
+      "b13",
+      "b18",
+      "b2",
+      "b21",
+      "b21a",
+      "b3",
+      "b31",
+      "b32",
+      "b37",
+      "b4",
+      "b47a",
+      "b6",
+      "cct1b",
+      "d1",
+      "d2",
+      "d3",
+      "d4",
+      "d5",
+      "d6",
+      "dst1b",
+      "e5",
+      "eff1",
+      "eff2",
+      "exc11",
+      "exc13",
+      "exc14",
+      "exc15",
+      "exc16",
+      "exc18",
+      "exc2",
+      "exc6",
+      "exc7",
+      "formal",
+      "ing4",
+      "it1",
+      "jc10",
+      "jc13",
+      "jc15a",
+      "jc16a",
+      "m1",
+      "mil10a",
+      "mil10e",
+      "pn4",
+      "pol1",
+      "pra8n",
+      "prot3",
+      "q2",
+      "q10a",
+      "q5b",
+      "r1",
+      "r12",
+      "r14",
+      "r15",
+      "r16",
+      "r18",
+      "r3",
+      "r4",
+      "r4a",
+      "redist1",
+      "redist2a",
+      "redist3",
+      "ros4",
+      "smedia1",
+      "smedia4",
+      "smedia7",
+      "vb10",
+      "vb2",
+      "vb50",
+      "vb58",
+      "vic1ext",
+      "w14a",
+      "wf1"
+    )
+
+dimensions <- c(
+      "a4",
+      "colorr",
+      "conocim",
+      "cp13",
+      "cp6",
+      "cp7",
+      "cp8",
+      "drk1",
+      "ed",
+      "env1calt",
+      "env2b",
+      "estratosec",
+      "exc7new",
+      "gi0n",
+      "idio2",
+      "idiomaq",
+      "infrax",
+      "inteval",
+      "l1",
+      "ocup1a",
+      "ocup4a",
+      "ocupoit",
+      "pais",
+      "pra10",
+      "pra2n",
+      "pra8an_1",
+      "pra8an_2",
+      "pra8an_3",
+      "pra8an_5",
+      "pra8an_6",
+      "pra8an_7",
+      "pra8an_77",
+      "psc1",
+      "psc10",
+      "psc11",
+      "psc11a",
+      "psc12",
+      "psc13_1",
+      "psc13_10",
+      "psc13_11",
+      "psc13_12",
+      "psc13_13",
+      "psc13_2",
+      "psc13_3",
+      "psc13_4",
+      "psc13_5",
+      "psc13_6",
+      "psc13_7",
+      "psc13_8",
+      "psc13_9",
+      "psc2",
+      "psc3_0",
+      "psc3_1",
+      "psc3_2",
+      "psc3_3",
+      "psc4",
+      "psc5",
+      "psc6",
+      "psc7",
+      "psc8",
+      "psc9",
+      "q1",
+      "q10e",
+      "q11n",
+      "q12bn",
+      "q12c",
+      "q14",
+      "q2",
+      "q3cn",
+      "q5a",
+      "r5",
+      "r6",
+      "r7",
+      "r8",
+      "smedia2",
+      "smedia3",
+      "smedia5",
+      "smedia6",
+      "smedia8",
+      "smedia9",
+      "soct2",
+      "ur",
+      "vb20",
+      "vb51",
+      "vb52",
+      "year"
+    )
+```
+
+
+```{r Save wide data as .csv}
+final_wide_measure <-
```

Comment:
This step does not execute properly--I get an error that "Values in `answer_measure` are not uniquely identified; output will contain list-cols." Most of the values in the resulting dataframe are lists, and I am unable to write out to a file. Any idea what could be going on?